### ISSUE TITLE
[1.12.2] Tinkers source code doesn't work in developing environment

### ISSUE THREAD

**Issue by delta-nos** [SlimeKnights/TinkersConstruct#4664]
### Minecraft Version

1.12.2

### Forge Version

14.23.5.2855

### Mantle Version

1.3.3.55

### Tinkers' Construct Version

2.13.0.179 or 2.13.0.183

### Describe your issue

i'm making tinkers' addon, after added source code in this github to my development library, runClient never run.
when my library has only Mantle, runClient worked successfully.
as far as i can see this log, are the versions different between mantle and tinkers?
however, i use latest code. am I right to use this code: https://github.com/SlimeKnights/TinkersConstruct/tree/1.12 ?

```
java.lang.IllegalAccessError: tried to access field net.minecraft.block.Block.displayOnCreativeTab from class slimeknights.mantle.block.BlockStairsBase
	at slimeknights.mantle.block.BlockStairsBase.<init>(BlockStairsBase.java:12)
	at slimeknights.tconstruct.common.TinkerPulse.registerBlockStairsFrom(TinkerPulse.java:72)
	at slimeknights.tconstruct.shared.TinkerCommons.registerBlocks(TinkerCommons.java:217)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_51_TinkerCommons_registerBlocks_Register.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus$1.invoke(EventBus.java:144)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
	at net.minecraftforge.registries.GameData.fireRegistryEvents(GameData.java:847)
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:630)
	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:252)
	at net.minecraft.client.Minecraft.init(Minecraft.java:467)
	at net.minecraft.client.Minecraft.run(Minecraft.java:378)
	at net.minecraft.client.main.Main.main(Main.java:118)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraftforge.legacydev.Main.start(Main.java:86)
	at net.minecraftforge.legacydev.MainClient.main(MainClient.java:29)
```

### Reproduced with only Tinkers' Construct?

Yes

### 

- [x] Confirm you have searched the tracker to see if your issue already exists
- [X] Confirm you are not using performance enhancers, such as optifine
- [X] Confirm you have tested without external core mods/mixins

**Comment by KnightMiner** [comment]
You set up your workspace wrong, so you are having access transformer issues. This problem is fixed in newer forge versions by simplifying access transformers. For 1.12, I never had the issue you are having.

In any case, realize 1.12 has not received new development since March 2020. A lot has changed in forge Gradle since then and we see no point in updating the old workspace when we are not releasing new builds. Only reason 1.12 still is receiving issue right now is so we can give support for non-code issues until 1.16 gets a beta

So in summary, workspace worked just fine when we last used it in 2020. You are on your own if you want to set it up again 2 years later. 

----

Also, you are not right trying to use the 1.12 branch if you are building an addon. Use maven to build addons. Most of my reply above assumed you were trying to build TiC from source as that is what your post first read as. You might have more luck talking with an author of a 1.12 addon, or try asking on discord

**Comment by xniki119x** [comment]
Maybe this will help, I had the same error, I added the string `public net.minecraft.block.Block field_149772_a # displayOnCreativeTab` in accesstransformer.cfg for mappings `stable_39`
My mod was built and Minecraft was running.

**Comment by KnightMiner** [comment]
1.12 is no longer supported. You are free to discuss support for it elsewhere, there is however no point replying to a 2 year old issue.
